### ISSUE TITLE
Add validation set and re-train.

### ISSUE THREAD

**Issue by moscmh** [moscmh/titanic#2]

**Comment by moscmh** [comment]
Still working on knn saturated model!

**Comment by moscmh** [comment]
Next - check if validation set reflects accurately by generating prediction on test set & upload.

**Comment by moscmh** [comment]
Next - 7th features at marked cell with date 20240726.

**Comment by moscmh** [comment]
Validation set created with stratified k folding.
